packages/context/src/context.ts

```diff
@@ -3,9 +3,9 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import debugFactory from 'debug';
+import debugFactory, {Debugger} from 'debug';
```

Comment:
> import debugFactory, {Debugger} from 'debug';

why not

import {debugFactory, Debugger} from 'debug';

Reply:
Please note `debug` module exports a function such as:

```js
module.exports = function debugFactory(...) {}
```

And `Debugger` is a member of the exports.

The TS imports reflect plain JS usage as follows:

```js
const debugFactory = require('debug'); // The module.exports is a function 
require('debug').Debugger; // debug is used as a namespace
```